src/rcc.rs

```diff
@@ -37,12 +47,36 @@ pub struct Rcc {
 
 #[allow(unused)]
 const HSI: u32 = 8_000_000; // Hz
+#[allow(dead_code)] // Only used for ADC, but no configuration for ADC clock yet.
+const HSI14: u32 = 14_000_000; // Hz - ADC clock.
```

Comment:
If we don't need it yet (and probably not in the future, since we do configuration of HSI14 in the ADC PR), i wouldn't add it

Reply:
Ok I will remove this then.